ext/sockets/tests/socket_recv_overflow.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+socket_recvfrom overflow on length argument
+--EXTENSIONS--
+sockets
+--FILE--
+<?php
```

Comment:
Test fails on Windows, also a bit of a weird indentation: normally we don't indent tests outer scope?

Reply:
yes that was just quick copy paste from my local test :)